CoreScriptsRoot/Modules/Stats/StatsAnnotatedGraph.lua

```diff
@@ -0,0 +1,6 @@
+--[[ Services ]]--
+local CoreGuiService = game:GetService('CoreGui')
```

Comment:
is this going to be updated at some point? would rather not add if not necessary